### PR TITLE
Automate Release to Ansible Galaxy

### PR DESCRIPTION
**Pull request by willtome**
This PR adds an automated GH action for releasing to Ansible Galaxy

- release.yml - playbook to create the collection and push to Galaxy
- galaxy.yml.j2 - template file for galaxy.yml file
- galaxy-release.yml - github workflow for triggered on release to run release playbook

Usage:
To utilize this workflow, create a secret for the `ANSIBLE_GALAXY_APIKEY` in the Settings section for the repository. In github, draft a new release with the tag version in the form `v.#.#.#`. This tag is used as the version # published to Galaxy. The workflow is triggered by clicking "Publish Release". Once it has complete, a new version of the collection will be available on Galaxy.

### CODE REVIEW

**Comment by willtome** [comment]
@sean-m-sullivan @jlozadad @sabre1041 @oybed

**Comment by jlozadad** [comment]
can you add a readme with the instructions you commented here? that way we can a somewat SOP of it.

File: README.md [hunk 1]
```diff
@@ -9,6 +9,16 @@
 
 This is a collection of roles for AWX/Tower.
 
+## Release Process
+This collection uses an auatomated GitHub workflow to publish releases to Ansible Galaxy. This workflow can be found in `.github/workflows/galaxy-release.yml`. It is dependent on `release.yml` and `galaxy.yml.j2`. See instructions below for usage.
```
Comment (jlozadad) [inline README.md:13]: typo here.

**Comment by jlozadad** [comment]
have to fix the conflict to merge in.

**Pull request merged**